### PR TITLE
Initialize

### PR DESCRIPTION
**Pull request by JasemDuncan**
For this part of the project, all configurations were made, also basic HTML structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,48 @@
-# bookstore
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Bookstore
```
Comment (TedLivist) [inline README.md:3]: - [x] Look at the image below. Your file structure should follow the template given in the project requirements. You are missing the `src/components` directory. Kindly add it.

  grace-file

File: src/redux/books/books.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                  <option value="Comedy">Comedy</option>
+                  <option value="Drama">Drama</option>
+                  <option value="Triller">Triller</option>
+                </select>
+              </div>
+            </div>
+            <div>
+              <input type="submit" value="Submit" />
+            </div>
+          </form>
```
Comment (TedLivist) [inline src/redux/books/books.js]: - [x]  In keeping with the SOLID principle, kindly extract this form adding a new book into a new component under the `src/components` folder.

File: src/redux/categories/categories.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      </>
+    );
+  };
+
+  return (
+    <Category />
+  );
+};
+
+export default Categories;
```
Comment (TedLivist) [inline src/redux/categories/categories.js]: - [x] The **`redux`** directory is meant for redux-related configurations, not components. At this point in the project, this file should be empty. Kindly move this **Categories** component into a new file in the `src/components` folder as that is where your components should live.

File: src/redux/books/books.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+const Books = () => {
+  const BooksData = {
+    gender: 'Action',
+    title: 'The Hunger Games',
+    director: 'Suzanne Collins',
+    porcentage: '64%',
+    chapter: 'Chapter 17',
+  };
+
+  const Book = () => {
```
Comment (TedLivist) [inline src/redux/books/books.js]: - [x] Kindly extract this **`Book`** component into a new component under `src/components`. Remember to separate the form from the Book component as, technically, they are independent of each other.
- [x] Your **`Book`** component should also take in the book data as a prop and render those data

File: src/redux/books/books.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+const Books = () => {
```
Comment (TedLivist) [inline src/redux/books/books.js]: - [x] The **`redux`** directory is meant for redux-related configurations, not components. At this point in the project, this file should be empty. Kindly move this **Books** component into the `src/components` folder as that is where your components should live.
- [x] The **`Books`** component is intended for listing books using the **`Book`** component and the component for adding a new book.

File: README.md [hunk 6]
```diff
@@ -1 +1,48 @@
...
+To get a local copy up and running follow these simple steps.
+
+### Setup
+ 
+
+### Install
+- Open your console and change the current working directory into the destination for this project.
+- Type `git clone git@github.com:JasemDuncan/bookstore.git`
+- Type Enter to finish the installation process.
+
```
Comment (TedLivist) [inline README.md]: - [x] Technically, users cannot run your app locally without running `npm i` and `npm start`. Kindly add these commands as necessary for setting up and running the application.

File: src/App.js [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+              <ul>
+                <li>
+                  <Link to="/">Books</Link>
+                </li>
+                <li>
+                  <Link to="/Categories">Categories</Link>
+                </li>
+              </ul>
+            </nav>
+          </div>
```
Comment (TedLivist) [inline src/App.js:36]: - [x] **[OPTIONAL]** To make our code look neater, I'd recommend that you extract this Navbar into a new component under `src/components` folder and render the component here instead.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @JasemDuncan,

Good job so far!
✔️ Nice documentation
✔️ Nice basic UI
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddBook.js [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+      id: 3,
+      name: 'Drama',
+    },
+    {
+      id: 4,
+      name: 'Triller',
+    },
+  ];
+
+  //   const [category, setCategory] = useState('');
```
Comment (rukundoeric) [inline src/components/AddBook.js]: - [x] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.

File: src/components/Book.js [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+import PropTypes from 'prop-types';
+
+const Book = (props) => {
+  const {
+    title, category,
+  } = props;
+
+  const BooksData = {
+    // gender: 'Action',
+    // title: 'The Hunger Games',
```
Comment (rukundoeric) [inline src/components/Book.js]: - [x] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.

File: src/components/Books.js [hunk 10]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+// import { useSelector } from 'react-redux';
```
Comment (rukundoeric) [inline src/components/Books.js]: - [x] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @JasemDuncan, 

_Great job so far 👏 💯 👍. You've done well implementing some of the requested changes. However, there are still some which aren't addressed yet. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 
  - Addressed some of the issues raised from the previous review. 💯 

## Required Changes ♻️
  - [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
  
  - N/A 

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @JasemDuncan, 

Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**